src/Lentille/GraphQL.hs

```diff
@@ -254,15 +256,18 @@ streamFetch client@GraphClient {..} mkArgs StreamFetchOptParams {..} transformRe
 
     -- Handle the response
     case respE of
-      Left e ->
+      Left e -> do
         -- Yield the error and stop the stream
-        S.yield (Left $ GraphError e)
+        now <- lift mGetCurrentTime
+        S.yield (Left $ GraphError now e)
```

Comment:
I guess here we should try to extract the pageInfo from the error to make the process continue, otherwise I'm not sure the current implementation will be able to resume after the error.